docs/src/main/sphinx/release/release-467.md

```diff
@@ -0,0 +1,105 @@
+# Release 467 (4 Dec 2024)
+
+## General
+
+* Change default protocol for internal communication to HTTP/1.1. ({issue}`24281`)
+* Fix query failures or wrong output in `SHOW STATS` when a connector returns
+  `NaN` values for table statistics. ({issue}`24315`)
+
+## Security
+
+## Web UI
+
+## JDBC driver
+
+## Server RPM
+
+## Docker image
+
+* Reduce image size and remove package manager.  ({issue}`24281`)
```

Comment:
@wendigo .. I think this is kinda needed. Wdyt @martint ? At least the package manager thing .. since it heavily impacts users who create derived docker images... it could even be considered a breaking change .. at least in my opinion

Reply:
You can still install packages using multi stage build approach (the same we've used). It's just a different approach. Maybe we can document that?

Reply:
Yes we should .. let me add this as a task to the packaging issue

Reply:
https://github.com/trinodb/trino/issues/22597

Reply:
Yeah, we should keep the "remove package manager" bit but remove the part about the image size.